arm/comm.go

```diff
@@ -672,7 +672,54 @@ func (x *xArm) MoveToPosition(ctx context.Context, pos spatialmath.Pose, extra m
 	if err := x.start(ctx); err != nil {
 		return err
 	}
-	if err := motion.MoveArm(ctx, x.logger, x, pos); err != nil {
+	point := pos.Point()
+	angles := pos.Orientation().AxisAngles()
+	c1 := x.newCmd(regMap["MoveToPos"])
+
+	floatBytes := make([]byte, 4)
+
+	binary.LittleEndian.PutUint32(floatBytes, math.Float32bits(float32(point.X)))
+	c1.params = append(c1.params, floatBytes...)
```

Comment:
still need to figure out how to pull this into its own function where it reuses both buffers the way it does here